k8s/charts/openebs/templates/daemonset-ndm.yaml

```diff
@@ -84,6 +84,8 @@ spec:
         - name: procmount
           mountPath: /host/proc
           readOnly: true
+        - name: basepath
+          mountPath: /var/openebs
```

Comment:
This is `/var/openebs/ndm` as mentioned in [ndm-operator.yaml](https://github.com/openebs/node-disk-manager/blob/3c858dd76bd4020d922be123c11cb6731a0d0dd1/ndm-operator.yaml#L142)

Reply:
Done